core/src/test/java/org/elasticsearch/search/query/SearchQueryIT.java

```diff
@@ -1892,17 +1892,18 @@ public void testIndicesQuery() throws Exception {
         assertSearchHits(searchResponse, "1", "2", "3");
         searchResponse = client().prepareSearch("index1", "index2", "index3")
                 .setQuery(indicesQuery(matchQuery("text", "value1"), "index1")
-                        .noMatchQuery("all")).get();
+                        .noMatchQuery(QueryBuilders.matchAllQuery())).get();
         assertHitCount(searchResponse, 3l);
         assertSearchHits(searchResponse, "1", "2", "3");
 
         searchResponse = client().prepareSearch("index1", "index2", "index3")
                 .setQuery(indicesQuery(matchQuery("text", "value1"), "index1")
-                        .noMatchQuery("none")).get();
+                        .noMatchQuery(QueryBuilders.matchNoDocsQuery())).get();
         assertHitCount(searchResponse, 1l);
         assertFirstHit(searchResponse, hasId("1"));
     }
 
+    @AwaitsFix(bugUrl = "//norelease we need to refactor nested queries for this test to pass")
```

Comment:
can we open an issue for this so we don't forget and link it here? I know that we will remember to get back to these queries but I also want to remember to re-enable this test ;)
also the norelease look weird within the bugUrl, I would take it out